spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewritePositionDeleteFilesAction.java

```diff
@@ -118,45 +117,45 @@ public TestRewritePositionDeleteFilesAction(
     this.format = format;
```

Comment:
the constructor should be removed and the format should be part of the parameters

Reply:
I don't think that would be right. 
The constructor is not only setting `format`, it also calls its parent's constructor which is calling TestBaseWithCatalog's constructor, which is starting the spark session and running sql queries.

When constructors are only setting the `@Parameter` values, I have removed the constructor. See my changes in `TestRewriteManifestsAction`.

Reply:
I just realized that it's a mistake to have any constructor in a parameterized base class. I've cleaned this up in https://github.com/apache/iceberg/pull/9368